include/RAJA/policy/tbb/scan.hpp

```diff
@@ -0,0 +1,158 @@
+/*!
+******************************************************************************
+*
+* \file
+*
+* \brief   Header file providing RAJA scan declarations.
+*
+******************************************************************************
+*/
+
+#ifndef RAJA_scan_sequential_HPP
```

Comment:
Ugh, indeed not.  I still need to do the scans, but TBB has optimized primitives for it so it should be a simple mapping.